Art Institute of Chicago.js

```diff
@@ -0,0 +1,222 @@
+{
+	"translatorID": "8e98b11a-5648-42b2-8542-5f366cb953f6",
+	"label": "Art Institute of Chicago",
+	"creator": "nikhil trivedi, Illya Moskvin",
+	"target": "^https?://(www.)?artic.edu/artworks/.*",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2021-02-18 21:51:25"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2021 Art Institute of Chicago
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc) {
+	if (ZU.xpathText(doc, '//html[contains(@class, "p-artwork-show")]')) {
+		return 'artwork';
+	}
+	return false;
+}
+
+function doWeb(doc) {
+	scrape(doc);
+}
+
+function scrape(doc) {
+	var item = new Zotero.Item('artwork');
+	item.title = ZU.xpathText(doc, '//h1');
+
+	var artists = ZU.xpath(doc, '//meta[contains(@name, "citation_author")]');
+	for (var i = 0; i < artists.length; i++) {
+		var cleaned = artists[i].content.replace(/\(.*\)$/, '').trim();
+		if (cleaned.split(' ').length > 2) {
+			item.creators.push({ lastName: cleaned, creatorType: 'artist', fieldMode: 1 });
+		}
+		else {
+			item.creators.push(ZU.cleanAuthor(cleaned, 'artist'));
+		}
+	}
+
```

Comment:
I think adding a snapshot would be appropriate here

Reply:
Sorry, I don't know what a snapshot is!